save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt

```diff
@@ -42,6 +42,14 @@ class VulnerabilityService(
     private val lnkVulnerabilityTagRepository: LnkVulnerabilityTagRepository,
     private val tagRepository: TagRepository,
 ) {
+    @Suppress("TYPE_ALIAS")
+    private val getTagMap: List<Vulnerability>.() -> Map<Vulnerability, Set<Tag>> = {
```

Comment:
I think you can try changing this to a private function

Reply:
I guess it is fine